x/oracle/keeper/keeper.go

```diff
@@ -41,17 +41,17 @@ type Keeper struct {
 	// PriceSnapshots maps types.PriceSnapshot to the asset.Pair of the snapshot and the creation timestamp as keys.Uint64Key.
 	PriceSnapshots   collections.Map[collections.Pair[asset.Pair, time.Time], types.PriceSnapshot]
 	WhitelistedPairs collections.KeySet[asset.Pair]
-	PairRewards      collections.IndexedMap[uint64, types.PairReward, PairRewardsIndexes]
+	PairRewards      collections.Map[uint64, types.Reward]
 	PairRewardsID    collections.Sequence
 }
 
 type PairRewardsIndexes struct {
 	// RewardsByPair is the index that maps rewards associated with specific pairs.
-	RewardsByPair collections.MultiIndex[asset.Pair, uint64, types.PairReward]
+	RewardsByPair collections.MultiIndex[asset.Pair, uint64, types.Reward]
```

Comment:
Does a `RewardsByPair` index even make sense anymore? I think we can get rid of this struct.

Reply:
Good point, I removed the logic but I totally forgot the hanging struct.